Source/WebCore/page/scrolling/ScrollingStateScrollingNode.h

```diff
@@ -38,6 +38,12 @@ OBJC_CLASS NSScrollerImp;
 
 namespace WebCore {
 
+struct MouseIsInScrollbar {
```

Comment:
`ScrollbarHoverState`. The current name makes it sound like it only applies to one scrollbar.

Reply:
Fixed.